src/zbBridgeAccessory.ts

```diff
@@ -32,6 +34,11 @@ export abstract class ZbBridgeAccessory {
     this.addr = this.accessory.context.device.addr;
     this.type = this.accessory.context.device.type;
     this.reachable = undefined;
+    this.endpoint = -1; 
+
+    if (this.accessory.context.device.endpoint !== undefined) {
```

Comment:
Not necessary if `undefined` is used - see above.

Reply:
Sure, my fault, I will change it.